### DISCUSSION TITLE
Confused about intial login

### DISCUSSION THREAD

**Discussion by githubjbassett** [mealie-recipes/mealie#2573] [Q&A]
I have attempted to install in a Docker container 3 times today, but I must be doing something wrong because I never have a username and password that will get me into the page.  I have tried the changeme@email.com, the root user, the db user, none work. I have tried initializing the database, but I get an error 'no module sqlalchemy'.  This is probably a simple fix, but I've read through the installation guide and followed them as well as I could.  Any suggestions would be great, thanks!

**Comment by michael-genson** [comment]
Have you followed the installation checklist? It sounds like the backend docker container isn't starting up correctly:
https://nightly.mealie.io/documentation/getting-started/installation/installation-checklist/

**Comment by githubjbassett** [comment]
The instructions were there, I just wasn't seeing them properly in context

**Reply by M1NiSTrY-GiT** [reply]
Could you please share what was the issue and how you fixed this? I'm having the exact same problem it seems. I'm on v1.0.0-RC1.1 with SQLite. Container starts fine, logs seem ok as well:

INFO: 25-Oct-23 10:06:59        Database connection established.
INFO: 25-Oct-23 10:06:59        Context impl SQLiteImpl.
INFO: 25-Oct-23 10:06:59        Will assume non-transactional DDL.
WARNING: 25-Oct-23 10:06:59     "get_all" method is deprecated; use "page_all" instead
INFO: 25-Oct-23 10:06:59        No food found with slug: 'dairy-products-and-dairy-substitutes' skipping fix
INFO:     Started server process [14]
INFO:     Waiting for application startup.
INFO: 25-Oct-23 10:07:02        -----SYSTEM STARTUP----- 

INFO: 25-Oct-23 10:07:02        ------APP SETTINGS------
INFO: 25-Oct-23 10:07:02        {
    "PRODUCTION": true,
    "BASE_URL": "https://xxx.xxx.xxx",
    "STATIC_FILES": "/spa/static",
    "IS_DEMO": false,
    "API_PORT": 9000,
    "API_DOCS": true,
    "TOKEN_TIME": 48,
    "LOG_LEVEL": "INFO",
    "GIT_COMMIT_HASH": "",
    "ALLOW_SIGNUP": true,
    "SECURITY_MAX_LOGIN_ATTEMPTS": 5,
    "SECURITY_USER_LOCKOUT_TIME": 24,
    "DB_ENGINE": "sqlite",
    "DB_PROVIDER": {
        "data_dir": "/app/data",
        "prefix": ""
    },
    "DEFAULT_GROUP": "Home",
    "DEFAULT_EMAIL": "xxx@xxx.xxx",
    "SMTP_HOST": null,
    "SMTP_PORT": "587",
    "SMTP_FROM_NAME": "Mealie",
    "SMTP_FROM_EMAIL": null,
    "SMTP_AUTH_STRATEGY": "TLS",
    "LDAP_AUTH_ENABLED": false,
    "LDAP_SERVER_URL": null,
    "LDAP_TLS_INSECURE": false,
    "LDAP_TLS_CACERTFILE": null,
    "LDAP_ENABLE_STARTTLS": false,
    "LDAP_BASE_DN": null,
    "LDAP_QUERY_BIND": null,
    "LDAP_QUERY_PASSWORD": null,
    "LDAP_USER_FILTER": null,
    "LDAP_ADMIN_FILTER": null,
    "LDAP_ID_ATTRIBUTE": "uid",
    "LDAP_MAIL_ATTRIBUTE": "mail",
    "LDAP_NAME_ATTRIBUTE": "name",
    "TESTING": false
}
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:9000 (Press CTRL+C to quit)
